src/app/containers/Video/index.jsx

```diff
@@ -16,9 +17,32 @@ const VideoContainer = ({ blocks }) => {
     return null;
   }
 
+  const nestedModel = deepGet(['model', 'blocks', 0, 'model'], aresMediaBlock);
+  const kind = deepGet(['subType'], nestedModel);
+  const pid = deepGet(['id'], nestedModel);
+  const title = deepGet(['title'], nestedModel);
+  const version = deepGet(['versions', 0], nestedModel);
+  const duration = deepGet(['duration'], version);
+  const versionID = deepGet(['versionId'], version);
+  const holdingImageUrl = deepGet(
+    ['blocks', 1, 'model', 'blocks', 0, 'model', 'locator'],
+    aresMediaBlock.model,
+  );
+  const items = {
```

Comment:
`items` should be an array that contains this object.

The reason it should be an array even though we're currently only setting one value, is so that in future we have the ability to pass in multiple items to a single media player. This is also the format that SMP takes in as part of its playlist JSON definition.

Reply:
Thanks for the explanation on this @sareh 👍